<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>md5Fix</class>
 <widget class="QDialog" name="md5Fix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Réparateur de liste noire</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dossier</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineFolder"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Noms de fichiers</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineFilename"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboSource"/>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonContinue">
       <property name="text">
        <string>Continuer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="radioForce">
     <property name="text">
      <string>Forcer le calcul du md5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="radioFilename">
     <property name="text">
      <string>Récupérer le md5 dans le nom du fichier</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineFolder</tabstop>
  <tabstop>radioForce</tabstop>
  <tabstop>radioFilename</tabstop>
  <tabstop>lineFilename</tabstop>
  <tabstop>comboSource</tabstop>
  <tabstop>buttonContinue</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioFilename</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioFilename</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineFilename</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
